CMakeLists.txt

```
@@ -8,10 +8,21 @@ cmake_minimum_required(VERSION 3.13.4)
 # Project configuration
 #-------------------------------------------------------------------------------
 
-project(iree-bare-metal-arm VERSION 1.0 LANGUAGES C)
+project(iree-bare-metal-arm VERSION 1.0 LANGUAGES C ASM)
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+option(BUILD_WITH_CMSIS "Builds the sample with CMSIS" OFF)
+option(BUILD_WITH_LIBOPENCM3 "Builds the sample with libopencm3" ON)
+
+if(BUILD_WITH_LIBOPENCM3 AND BUILD_WITH_CMSIS)
```

```suggestion
if(BUILD_WITH_CMSIS AND BUILD_WITH_LIBOPENCM3)
```
Sort options alphabetically as in the other code.